react/features/mobile/navigation/components/RootNavigationContainer.js

```diff
@@ -18,7 +18,7 @@ import {
 
 import ConferenceNavigationContainer
     from './conference/components/ConferenceNavigationContainer';
-import LoadConfigOverlay from './overlay/components/LoadConfigOverlay';
+import ConnectingPage from './connecting/components/ConnectingPage';
```

Comment:
Don't create a directory just for the ConnnectingPage component, just put it here, in the navigation/components.